decentraland/unity-explorer
# feat: beta allow list

## What does this PR change?

Adds the feature: https://www.notion.so/decentraland/Explorer-Alpha-Allow-list-for-private-beta-a883b197ced248d2befe13323a855308

Dependant of https://github.com/decentraland/unity-explorer/pull/1279 (the feature flag system is here too, but it would be ideal to simplify this pr by merging the previous one first)

Feature flag configuration: https://features.decentraland.systems/#/features/variants/explorer-user-allow-list

## How to test the changes?

1. Request access to the beta client to @lorux0 
2. Start the authentication flow without any changes, until you enter in-world
3. Change your account
4. In the authentication flow you will be requested to apply to the beta
5. You should not be able to enter to the world in any way

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
